docs/source/api_reference.rst

```diff
@@ -219,6 +219,19 @@ Email and URL Transform Primitives
     URLToTLD
 
 
+Expanding Transform Primitives
```

Comment:
I'm not exactly clear on the best way to organize all this, but these are primarily being added for time series problems, so it might make sense to add them to the Time Series Transform Primitive section instead of creating a new one here.